lib/lambda_ethereum_consensus/p2p/gossipsub.ex

```diff
@@ -19,9 +19,8 @@ defmodule LambdaEthereumConsensus.P2P.GossipSub do
       {"voluntary_exit", SszTypes.SignedVoluntaryExit},
       {"proposer_slashing", SszTypes.ProposerSlashing},
       {"attester_slashing", SszTypes.AttesterSlashing},
-      {"bls_to_execution_change", SszTypes.SignedBLSToExecutionChange}
-      # use type SignedAggregateAndProof
-      # {"beacon_aggregate_and_proof", SszTypes.SignedBeaconBlockHeader},
+      {"bls_to_execution_change", SszTypes.SignedBLSToExecutionChange},
+      {"beacon_aggregate_and_proof", SszTypes.SignedAggregateAndProof}
```

Comment:
this is nice, but should we subscribe to these topics if we're not really processing them?

Reply:
I left them to try out the SSZ library, and the most common gossip we are receiving is from the `beacon_aggregate_and_proof` topic. Should I comment/remove them?

Reply:
I guess we can leave it for now so that we can test that we're actually receiving things.